### PR TITLE
Adds related partymap to AnimateMap

### PR DESCRIPTION
**Pull request by mike-lvov**
Adds related partymap to AnimateMap
closes https://github.com/sparkletown/internal-sparkle-issues/issues/1063
closes https://github.com/sparkletown/internal-sparkle-issues/issues/1064

Related to https://github.com/sparkletown/internal-sparkle-issues/issues/931

### CODE REVIEW

File: src/components/templates/AnimateMap/AnimateMap.tsx [hunk 1]
```diff
@@ -28,6 +29,10 @@ export const AnimateMap: React.FC<AnimateMapProps> = ({ venue }) => {
   const store = useStore();
   const user = useUser();
 
+  // TODO: Use this data to display venues
+  const rooms = useRelatedPartymapRooms({ venue });
+  console.log(rooms);
```
Comment (alexey-petrov) [inline src/components/templates/AnimateMap/AnimateMap.tsx:34]: let's remove it

Comment (mike-lvov) [reply src/components/templates/AnimateMap/AnimateMap.tsx]: done by @yarikoptic

**Comment by whoaitsaimz** [comment]
Since this was done for all `staging` I'd like a bit more insight about how this "connection" is made. Will ping on the Port-a-palooza issue

File: src/components/templates/AnimateMap/AnimateMap.tsx (hunk 1 above)
Comment (yarikoptic) [reply src/components/templates/AnimateMap/AnimateMap.tsx]: Oh, was I useful this Friday? Cool

Comment (mike-lvov) [reply src/components/templates/AnimateMap/AnimateMap.tsx]: @yarikoptic ahaha, sorry. Tagged the wrong person. Have great weekends!

**Pull request merged**